src/main/java/mateAcademy/dao/impl/TestDaoImpl.java

```diff
@@ -0,0 +1,33 @@
+package mateAcademy.dao.impl;
+
+import mateAcademy.dao.AbstractDao;
+import mateAcademy.dao.TestDao;
+import mateAcademy.test.Test;
+
+import javax.persistence.EntityManager;
+import java.time.LocalDate;
+import java.util.List;
+
+public class TestDaoImpl extends AbstractDao<Test, Long> implements TestDao {
+
```

Comment:
empty line between class declaration and constructor or method or field should be deleted everywere were you have written code in such way